keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/ImageService.kt

```diff
@@ -98,10 +98,10 @@ class ImageService(
    * Each ami must have tags.
    */
   suspend fun getLatestNamedImageWithAllRegionsForAppVersion(appVersion: AppVersion, account: String, regions: List<String>): NamedImage? =
-    cloudDriverService.images(
+    cloudDriverService.namedImages(
       serviceAccount = DEFAULT_SERVICE_ACCOUNT,
-      provider = "aws",
-      name = appVersion.toImageName().replace("~", "_")
+      imageName = appVersion.toImageName().replace("~", "_"),
+      account = account
```

Comment:
Could this end up being the account we'll want to deploy to instead of the account containing the image?

Reply:
We pass in "test" here (when used in the ImageHandler). Same with the use in the ImageResolver. So I don't think so.